ajax/libs/tent-css/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "tent-css",
+  "filename": "tent.min.css",
+  "description": "A CSS survival kit.",
+  "author": "Aaron Mazade <ulinaaron@gmail.com>",
+  "license": "MIT",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/sitetent/tentcss.git"
+  },
+  "homepage": "https://css.sitetent.com",
+  "keywords": [
+    "css",
+    "flexbox",
+    "html",
+    "html5",
+    "scss",
+    "sass",
+    "responsive",
+    "framework"
+  ],
+  "npmName": "tent-css",
+  "npmFileMap": [
+    {
+      "basePath": "dist",
+      "files": [
+        "!(06_components){,**/*}"
```

Comment:
What is the intention of this glob?

Reply:
I don't think we support that

Reply:
In old versions there is folder in dist called 06_components
[https://github.com/sitetent/tentcss/tree/v1.1.1/dist](https://github.com/sitetent/tentcss/tree/v1.1.1/dist)

Reply:
I'm not sure glob syntax like this will work -- maybe best to add it to the .gitignore?